### DISCUSSION TITLE
Send magnet links to Vuetorrent?

### DISCUSSION THREAD

**Discussion by hundredfire** [VueTorrent/VueTorrent#1963] [Q&A] [answered]
Hello guys! 
I'm not sure if I'm asking in the right place, but do you have a solution to directly send a magnet link from a website straight to Vuetorrent (I'm running a qbittorrent docker container)? Maybe a browser extension?

**Comment by Larsluph** [comment, marked as answer]
PWA supports magnet URL handling. It's the install prompt you have when you arrive in the page for the first time.

**Comment by hundredfire** [comment]
Thanks. Not sure how I missed that
